doma-kotlin/src/main/kotlin/org/seasar/doma/kotlin/jdbc/criteria/statement/KUnifiedInsertStating.kt

```diff
@@ -0,0 +1,32 @@
+package org.seasar.doma.kotlin.jdbc.criteria.statement
+
+import org.seasar.doma.jdbc.criteria.context.SubSelectContext
+import org.seasar.doma.jdbc.criteria.statement.UnifiedInsertStarting
+import org.seasar.doma.kotlin.jdbc.criteria.declaration.KInsertSelectDeclaration
+import org.seasar.doma.kotlin.jdbc.criteria.declaration.KValuesDeclaration
+import java.util.*
+
+class KUnifiedInsertStating<ENTITY : Any>(private val statement: UnifiedInsertStarting<ENTITY>) {
```

Comment:
I found typo.

x: KUnifiedInsertStating
o: KUnifiedInsertStarting

The following class is the same.

org.seasar.doma.kotlin.jdbc.criteria.statement.KUnifiedDeleteStating
org.seasar.doma.kotlin.jdbc.criteria.statement.KUnifiedInsertStating
org.seasar.doma.kotlin.jdbc.criteria.statement.KUnifiedSelectStating
org.seasar.doma.kotlin.jdbc.criteria.statement.KUnifiedUpdateStating

Reply:
Good catch! Thanks!